css/style.css

```diff
@@ -129,3 +127,115 @@ ul {
     cursor: pointer;
 }
 
+
+
+/* Block banner */
+.banner {
+    height: 743px;
```

Comment:
it's not a good idea to set height for elements. It's better to achieve the same result using `paddings` and `margins`
[See details here](https://doka.guide/css/height/#:~:text=%D0%A2%D0%B0%D0%BA%D0%B6%D0%B5%20%D0%BE%D0%BD%D0%BE%20%D0%BC%D0%BE%D0%B6%D0%B5%D1%82%20%D0%BF%D1%80%D0%B8%D0%B3%D0%BE%D0%B4%D0%B8%D1%82%D1%81%D1%8F%20%D0%BF%D1%80%D0%B8%20%D0%B0%D0%B4%D0%B0%D0%BF%D1%82%D0%B8%D0%B2%D0%BD%D0%BE%D0%B9%20%D0%B2%D1%91%D1%80%D1%81%D1%82%D0%BA%D0%B5.)